common/src/test/java/org/corfudb/common/util/CompatibilityVectorUtilsTest.java

```diff
@@ -0,0 +1,50 @@
+package org.corfudb.common.util;
+
+import com.google.protobuf.ByteString;
+import org.corfudb.common.util.CompatibilityVectorUtils.Feature;
+import org.junit.Ignore;
+import org.junit.Test;
```

Comment:
Import of Ignore could be removed I think

Reply:
Done. Thank you.